wire-ios/Wire-iOS Tests/ConversationList/Container/ConversationListViewControllerSnapshotTests.swift

```diff
@@ -84,6 +85,14 @@ final class ConversationListViewControllerTests: XCTestCase {
             selfProfileViewControllerBuilder: .mock
         )
         sut.mainSplitViewState = .collapsed
+        // sut.navigationController?.navigationBar.barTintColor = SemanticColors.View.backgroundDefaultWhite
```

Comment:
To delete?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/2c669b9a98137eb0ce0aa305e9e3d9731d5bed5d